roaster-bom/pom.xml

```diff
@@ -11,11 +11,12 @@
 
 	<properties>
 		<version.org.eclipse.microprofile>6.0</version.org.eclipse.microprofile>
-		<version.hu.icellmobilsoft.coffee>2.3.0</version.hu.icellmobilsoft.coffee>
+		<version.hu.icellmobilsoft.coffee>2.7.0-SNAPSHOT</version.hu.icellmobilsoft.coffee>
```

Comment:
Ezt egyeztetni kéne hogy 2.7.0-t mikor akarunk kiadni, meg akkor a roasterből is, hogy ne legyen probléma a kiadásból.